packages/utils/test/retry.test.ts

```diff
@@ -122,4 +122,48 @@ describe('Retry', () => {
 
     expect(after).toBeGreaterThanOrEqual(now + 500);
   });
+
+  it('delay increases exponentially if using exponentialBackoff', async () => {
+    jest.useFakeTimers('modern');
+    jest.setSystemTime(0);
+
+    const throwFn = jest.fn().mockImplementation(() => {
+      throw new Error(`threw`);
+    });
+
+    retry(throwFn, {
+      exponentialBackoffDelay: 1000,
+      maxExponentialBackoffDelay: 7000,
+    })();
+
+    // Should call immediately
+    expect(throwFn).toHaveBeenCalledTimes(1);
+
+    // Exponential backoff should only call a second time after 2000ms
```

Comment:
This makes me realize that if you want to have back off like: 1 second, 2 seconds, 4 seconds, 8 seconds, then you need to set the `exponentialBackoffDelay` to 500 ms.